frontend/src/Components/AdminUpdateProduct.js

```diff
@@ -87,26 +86,30 @@ const AdminUpdateProduct = (props) => {
               <input
                 required
                 value={form.price}
+                type="number"
+                min="0"
+                step="0.01"
                 onChange={(e) => {
                   setForm({
                     ...form,
                     price: e.target.value,
                   });
                 }}
               />
-            </div>
-            <div>
-              <label> Image URL </label>
+              </div>
+              <div>
+              <label> Image </label>
               <input
-                value={form.image}
+                required
+                type="file"
+                accept="image/*"
                 onChange={(e) => {
-                  setForm({
-                    ...form,
-                    image: e.target.value,
-                  });
+                  setImage(e.target.files[0]);
                 }}
               />
```

Comment:
If you nest an `<input>` inside its `<label>` then screenreaders will automatically associate the two, and clicking the label will focus the input:

```js
            <label>
              Image 
              <input
                required
                type="file"
                accept="image/*"
                onChange={(e) => {
                  setImage(e.target.files[0]);
                }}
              />
            </label>
``

Reply:
Applied this change 👍